includes/admin/class-wcmp-export.php

```diff
@@ -1405,6 +1350,103 @@ public static function saveTrackTracesToOrders(MyParcelCollection $collection, a
             WCMP_Export::addTrackTraceNoteToOrder($order_id, $trackTraces);
         }
     }
+
+    /**
+     * @param WC_Order $order
+     *
+     * @return string
+     */
+    public static function getShippingName(WC_Order $order): string
+    {
+        if (method_exists($order, "get_formatted_shipping_full_name")) {
+            return $order->get_formatted_shipping_full_name();
+        }
+
+        return trim($order->get_shipping_first_name() . " " . $order->get_shipping_last_name());
+    }
+
+    /**
+     * @param WC_Order $order
+     * @param bool     $isUsingMyParcelFields
+     *
+     * @return array
+     * @throws Exception
+     */
+    public static function processPGAddress(WC_Order $order, bool $isUsingMyParcelFields): array
```

Comment:
afkortingen schrijven we niet in "full hoofdletters", dan wordt het `processPgAddress`. En ik ben uberhaupt wel tegen afkortingen maar ik snap het want "pakjegemak" is ook wel vies. :P

Reply:
Oke, zal ik even aanpassen dan :)